guide/onboarding/automatic-cloud-backup.md

```diff
@@ -9,19 +9,21 @@ permalink: /guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup
 main_classes: -no-top-padding
 ---
 
-# Automatic cloud backup
+# Cloud backup
 
 ### How it works
 
-The user’s key is encrypted and backed up on their respective device/cloud storage provider. If their device is lost, stolen, or broken, the user can log into their cloud storage account (such as Apple iCloud or Google Drive) to regain access to their funds. To learn more about the technical details, hop over to the [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) section on this topic.
+The user's wallet recovery data and channel state are backed up on their respective cloud storage provider. The recovery phrase only needs to be backed up once. However, the channel state may need to be backed up frequently because it will change every time the user's wallet sends, receives, or routes a payment; therefore, the channel state should be backed up automatically.
+
+If the user's device is lost, stolen, or broken, they can use a backup from their cloud storage account (such as Apple iCloud or Google Drive) to regain access to their funds. To learn more about the technical details, hop over to the [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) section on this topic.
 
 > Crypto rookies perceive themselves as “incapable of self-managing keys”.
 >
 > <cite>As researched by <a href="https://voskart.de/pdf/u_in_crypto.pdf">Abramova et al</a></cite>
 
 This method provides users with the feeling that they can rely on their cloud provider to regain access to their funds rather than putting the responsibility on themselves.
 
-Below is an example of what the user interface might look like for an automatic cloud backup that significantly reduces friction during onboarding.
+Below is an example of what the user interface might look like for an automatic cloud backup that significantly reduces friction during onboarding. An optimal phase to show this may be after the wallet has received funds for the first time, so that the user has incentive to back up their wallet.
```

Comment:
May want to add one more image to this carousel with a wallet with some funds in it and a prompt to backup now that funds are in the wallet.

Reply:
Done